meson.build

```diff
@@ -1,7 +1,9 @@
 project(
     'vs-fltk',
     ['c', 'cpp', 'swift'],
-    default_options: ['c_std=gnu23', 'cpp_std=gnu++23'],
+    version: '0.1.0-dev',
+    meson_version: '>= 1.1',
+    default_options: ['c_std=gnu2x', 'cpp_std=gnu++23'],
```

Comment:
meson.build:1:0: ERROR: None of values ['gnu23'] are supported by the C compiler. Possible values for option "C_std" are ['none', 'c89', 'c99', 'c11', 'c17', 'c18', 'c2x', 'gnu89', 'gnu99', 'gnu11', 'gnu17', 'gnu18', 'gnu2x']

Reply:
I think you are using an "outdated" compiler. I am making explicit use of the `#embed` preprocessor directive which has been set standard with C23 (and as such covered by gnu23). Any prior version is unlikely to work.
On my system and on the flatpak I am using clang-19 for that reason.  You might be able to also use very recent zig (it is just an updated clang frontend in C/C++ mode).  

If you look at `project.json`, 
```
"meson-setup.clang-release": "meson setup --reconfigure build/ --buildtype=release --native-file toolchains/clang.ini"
```
I am using a specific native file to ensure that c23 is supported.
This is briefly discussed in: https://github.com/KaruroChori/vs-fltk/blob/master/docs/for-developers.md

Reply:
Understood. I made some changes. Please trigger the workflow when it's convenient.